source/globalCommands.py

```diff
@@ -1534,6 +1534,20 @@ def script_toggleFocusMovesNavigatorObject(self,gesture):
 	script_toggleFocusMovesNavigatorObject.__doc__=_("Toggles on and off the movement of the navigator object due to focus changes") 
 	script_toggleFocusMovesNavigatorObject.category=SCRCAT_OBJECTNAVIGATION
 
+	def script_toggleBrowseMovesFocus(self,gesture):
+		if config.conf["virtualBuffers"]["focusFollowsBrowse"]:
+			# Translators: presented when toggled.
+			state = _("Fast browse mode")
+			config.conf["virtualBuffers"]["focusFollowsBrowse"]=False
+		else:
+			# Translators: presented when toggled.
+			state = _("Traditional browse mode")

```

Comment:
I would prefer that these messages reflected similar wording to what is in the Settings dialog. I.e. Focus following browse mode, or focus not following browse mode.   "Fast / traditional" is a little mis-leading I think :)